File: app/src/main/java/cn/intret/app/picgo/model/event/ExcludeFolderListChangeMessage.java
package cn.intret.app.picgo.model.event;


public class ExcludeFolderListChangeMessage extends BaseMessage {

    public ExcludeFolderListChangeMessage(String desc) {
        super(desc);
    }

    public ExcludeFolderListChangeMessage() {
    }

    @Override
    public String toString() {
        return "ExcludeFolderListChangeMessage{} " + super.toString();
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/event/ExcludeFolderListChangeMessage.kt
package cn.intret.app.picgo.model.event


class ExcludeFolderListChangeMessage : BaseMessage {

    constructor(desc: String) : super(desc)

    constructor()

    override fun toString(): String {
        return "ExcludeFolderListChangeMessage{} " + super.toString()
    }
}
